integration/tests/test_sabre.rs

```diff
@@ -60,7 +60,7 @@ impl StdError for TestError {
         }
     }
 
-    fn cause(&self) -> Option<&StdError> {
```

Comment:
Another solution to this would be to remove most of these `cause` methods, as they are equivalent to  the default implementation.

Reply:
Updated the commit to only include those that are not the default. The rest will be removed in https://github.com/hyperledger/sawtooth-sabre/pull/66